docs-user/guide-ui-tour.md

```diff
@@ -1,57 +1,105 @@
+![A screenshot highlighting the commit ranges in the top toolbar.](images/ui-tour-ranges.png)
 
-Committing a range selection by clicking the zoom button is a useful way to zoom in on a useful selection. It can be helpful to commit several ranges to hone in on specific parts of a profile.
+Committing a range selection by clicking the zoom button is a useful way to zoom in on a useful selection. It can be helpful to commit several ranges to focus on specific parts of a profile.
 
-## Timeline's thread stack graph
+## The timeline's screenshots
 
-![A screenshot highlighting the thread stack graph.](images/ui-tour-thread-stack.png)
+![A screenshot highlighting the screenshot track in the timeline.](images/ui-tour-timeline-screenshots.png)
 
-Each thread in the timeline contains a thread stack graph. The X axis represents time, and the Y axis represents the height of the stack. It's important to realize that the height does not correlate with the amount of work that was being done. It only represents the arbitrary height of the stack at the time. It can be used as a useful way to get oriented with what was executing at the time.
+The screenshots track help navigate in the profile and focus on the part that are
+more interesting to analyze.
 
-Clicking on the thread stack graph will select that stack and update the panel below (if the panel uses the currently selected stack). For instance, in the call tree, it will open up the tree to find the stack. The selected stack will be colored a darker blue in the list. This is another useful way to correlate stacks with when they were executed over time.
+## Timeline's activity graph
 
-[Stacks and samples can be filtered out ](./guide-filtering-call-trees.md) through various operations. When this happens, the thread stack graph will be empty at that spot. When the thread didn't have samples at all for an area (for instance the thread hadn't yet started up), then it should have an indicator as to the reason why there are no samples.
+![A screenshot highlighting the activity graph.](images/ui-tour-activity-graph.png)
+
+Each thread in the timeline contains an activity graph. The X axis represents time, and the Y axis represents the CPU usage.
+
+Clicking on the activity graph will select a sample and update the panel below (if the panel uses the currently selected stack). For instance, in the call tree, it will open up the tree to find the stack for this sample. The selected stack will be colored a darker blue in the list. This is another useful way to correlate stacks with when they were executed over time.
+
+[Stacks and samples can be filtered out ](./guide-filtering-call-trees.md) through various operations. When this happens, the activity graph will be greyed at that spot. When the thread didn't have samples at all for an area (for instance the thread hadn't yet started up), then it should have an indicator as to the reason why there are no samples.
```

Comment:
super nit: extraneous space at the end of the link 😄

Reply:
heh it was there before :) but I removed it